src/csharp/Extensions/Microsoft.Spark.Extensions.FileSystem.E2ETest/Constants.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.Spark.Extensions.FileSystem.E2ETest
+{
+    /// <summary>
+    /// Constants related to the FileSystem test suite.
+    /// </summary>
+    internal class Constants
```

Comment:
Is this file needed? What else are we expecting to be added here?

Reply:
This is the pattern we're using in all the test projects.